src/vanilla/mod.rs

```diff
@@ -0,0 +1,35 @@
+use std::env;
+use std::path::PathBuf;
+
+fn var(key: &str) -> String {
+    match env::var(key) {
+        Ok(val) => val,
+        Err(err) => panic!("couldn't find env var {}; err: {:?}", key, err)
```

Comment:
I'd say `io::Result`, but then again I don't know what we need `root_path` for in the first place.